docs/endpoints/info/bidders/allBidders.md

```diff
@@ -0,0 +1,75 @@
+# Prebid Server Bidders
+
+## `GET /info/bidders/all`
+
+This endpoint return details about all the bidders supported by prebid-server.
```

Comment:
Could add a line about how this is the same information that the `/info/biders/{bidderName}` returns, but just collects all the bidders into one request.